src/kernels/jupyter/interpreter/jupyterInterpreterService.node.ts

```diff
@@ -112,6 +115,17 @@ export class JupyterInterpreterService {
             // Use current interpreter.
             interpreter = await this.interpreterService.getActiveInterpreter(undefined);
             if (!interpreter) {
+                if (err) {
```

Comment:
I read this once or twice and honestly it still looks a bit off so I'll just ask. I'm not sure that I understand the flow here. Per my reading of the bug this issue is for when there is a selected Jupyter interpreter, but for when something happens to that Jupyter interpreter which makes it fail to start. So in that case getSelectedInterpreter would still have the selected jupyter interpreter and it wouldn't go down this path at all.

Reply:
This seem restricted to a very odd / unlikely case where we have an error, but we also don't have a selected Jupyter extension or an active python interpreter.

Reply:
here's the work flow,
* user selects an en
* deletes the env as per issue
* now they get a quick pick without any error message


this charge merely displays the error message

Reply:
ie workflow is better now, rather than displaying a quick pick for no reason, which would be very weird for the user. now the quick pick is after a user action (button click) from the prompt

Reply:
I was under the impression that the getActiveInterpreter right above this almost always returns with something unless there is no python detected at all on the system. Is that not the case?

Reply:
nope, because the active interpreter happened to be the same interpreter that was used for jupyter